### ISSUE TITLE
SmallWorld2007.2.src.zip is also a web page

### ISSUE THREAD

**Issue by Kirtai** [kyle-github/littlesmalltalk#3]
The file SmallWorld2007.2.src.zip contains a web page, not an archive.

**Comment by kyle-github** [comment]
Thanks for noticing this!  I got these files from someone else.   I will search around to see if I can find the original Java sources.

**Comment by kyle-github** [comment]
I found a version from 2004, but it looks like SmallWorld2007 is from Russell Allen's site.  I'll see if I can find something on the Wayback machine (the Internet archive).

**Comment by kyle-github** [comment]
I am not sure where I got the original SmallWorld2007 file.   That was definitely a page from Russell's web site.   Russell now has [SmallWorld](https://github.com/russellallen/SmallWorld) on GitHub.  It looks like the last version was from 2015.  The previous version was from 2007.

I added Budd's original SmallWorld to the archive directory and removed Russell's version.  I added a note about Russell's version to the README.

Thanks again for noticing this problem!
